gazebo_ros/test/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ set(tests
   test_gazebo_ros_properties
   test_gazebo_ros_state
   test_node
+  test_node_lookup
```

Comment:
Maybe we can call this ``test_reject_duplicate_nodes``

Reply:
okay.